### ISSUE TITLE
Measure tool not available when part is below build plate

### ISSUE THREAD

**Issue by bigwoof** [SoftFever/OrcaSlicer#5904]
### Is there an existing issue for this problem?

- [X] I have searched the existing issues

### OrcaSlicer Version

2.1.1

### Operating System (OS)

macOS

### OS Version

14.5 (23F79)

### Printer

Generic - Ender3 clone

### How to reproduce

1. add stl part onto build plate
2. move part below build plate (intention is to only print form a certain height upwards - it is a quick and dirty way to modify models when you are too lazy to modify at source)
3. press "U" to invoke measure tool - it and it does not work (greyed out)  

### Actual results

Before moving part:
![image](https://github.com/SoftFever/OrcaSlicer/assets/880522/8a422342-d86d-4a45-90ba-8bda42c33c8a)
After moving part:
![image](https://github.com/SoftFever/OrcaSlicer/assets/880522/75587813-66b8-4e94-bebd-21c9f6974acc)

### Expected results

I think that the slicer should still be able to measure parts when they are clipped by the build plate. 

### Project file & Debug log uploads

[needle holder.3mf.zip](https://github.com/user-attachments/files/16021982/needle.holder.3mf.zip)
[logs.zip](https://github.com/user-attachments/files/16022000/logs.zip)

### Checklist of files to include

- [X] Log file
- [X] Project file

### Anything else?

Loving the slicer and continuous development with the latest features!

**Comment by sleewok** [comment]
I can confirm that this is an issue. I am also encountering this.

**Comment by fritzw** [comment]
The most annoying part is that this occurs if *any* part on *any* build plate is below the surface, and it does not even tell you why the button is disabled. I was very confused about this yesterday when working on a large project file, and only found the cause just now after I accidentally moved an object down and noticed the Measure button disabling itself.

**Comment by fritzw** [comment]
Not stale.

**Comment by sleewok** [comment]
Definitely not

**Comment by sleewok** [comment]
bump

**Comment by fritzw** [comment]
@sleewok I just tested it in 2.3.0 on Linux and could not reproduce the issue. Does it still exist for you? Otherwise it should be closed.

**Comment by sleewok** [comment]
> [@sleewok](https://github.com/sleewok) I just tested it in 2.3.0 on Linux and could not reproduce the issue. Does it still exist for you? Otherwise it should be closed.

Good catch! I just tested again on 2.3.1 and it does appear to be fixed. Looks like it can be closed.
